### ISSUE TITLE
Expanding extending-modx/development-environment

### ISSUE THREAD

**Issue by Ibochkarev** [modxorg/Docs#102]
## Affected page(s)

https://docs.modx.org/current/en/extending-modx/development-environment

## What needs to be improved?

This documentation needs to be (re)written as the previous version was not useful. The Building an Extra in MODX Revolution is a good start, explaining the different concepts in tutorial form.

**Comment by Ibochkarev** [comment]
Related https://docs.modx.org/current/en/contribute/code/development-environment

**Comment by ant-tar** [comment]
I can try extend page description if issue is still free  :)

**Comment by Mark-H** [comment]
Yes, it still is :) There's still a lot of bounties available on documentation issues.
